### PR TITLE
Somrius

### PR DESCRIPTION
**Pull request by x3bo**
I just add a new adventure.

Please review this, @casey-collab

### CODE REVIEW

**Comment by shersom** [comment]
@casey-collab, I think @x3bo wanted this auto-merged.

File: catala/somrius/somrius.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Somrius. Content i feliç com quan eres un nen.
+
+El somni de la teva vida de trobar una casa de núvols de sucre, com el d'en Hansel i la Gretel, s'ha complert!
+
+Prens un bocí de núvol i recordes que vols volar. unes ales t'apareixen a l'esquena. Ara tens un dilema, què fas?
+
+[Continues menjant núvols de sucre](create-your-own-adventure/catala/menjar/menjar.md)
```
Comment (casey-collab) [inline catala/somrius/somrius.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: catala/somrius/somrius.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+[Surts per la finestra i alces el vol](create-your-own-adventure/catala/somrius/icar/icar.md)
```
Comment (casey-collab) [inline catala/somrius/somrius.md]: This link is also invalid.

File: catala/somrius/icar/icar.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+Emprens el vol, com un ocell!
```
Comment (casey-collab) [inline catala/somrius/icar/icar.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: catala/somrius/icar/icar.md (hunk 3 above)
Comment (x3bo) [inline catala/somrius/icar/icar.md]: @casey-collab, If the link below works, you can reach this page

**Comment by x3bo** [comment]
@casey-collab I think I update it correctly. Please, merge it when you can. Thanks.


And thank you too, @shersom, for yor help!! Your advice has been so useful!

**Comment by casey-collab** [comment]
Looks great! :heavy_check_mark:

**Comment by shersom** [comment]
Awesome! Glad this finally worked!

**Pull request merged**